encrypted-media/scripts/check-encryption-scheme.js

```diff
@@ -0,0 +1,42 @@
+function runTest(config, qualifier)
+{
+  function checkEncryptionScheme(encryptionScheme)
+  {
+    var simpleConfig = getSimpleConfiguration();
+    assert_greater_than(simpleConfig[0].audioCapabilities.length, 0);
+    simpleConfig[0].audioCapabilities.forEach(function(capability) {
+      capability.encryptionScheme = encryptionScheme;
+    });
+
+    return navigator.requestMediaKeySystemAccess(config.keysystem, simpleConfig)
+        .then(
+            function(access) {
+              var actualConfiguration = access.getConfiguration();
+              for (let i = 0; i < actualConfiguration.audioCapabilities.length; i++) {
+                const capability = actualConfiguration.audioCapabilities[i];
+
+                // If "encryptionScheme" is not supported, fail.
+                if (!('encryptionScheme' in capability)) {
+                  return Promise.reject('Not implemented');
+                }
+
+                // If "encryptionScheme" is supported, it should be returned.
+                assert_equals(capability.encryptionScheme, encryptionScheme);
+              }
+              return Promise.resolve('Supported');
+            },
+            function error() {
+              // CDM does not support "encryptionScheme". Test should
+              // still pass.
```

Comment:
nit: does this fit in one line?

Reply:
This was done by the Chromium formatter. Fixed.